load_balancer/test/statusTest.js

```diff
@@ -103,7 +103,7 @@ describe('Method selectActiveNodes returns expected array of nodes', () => {
 
   it('Correct array for an input array of working nodes', () => {
     nodes = nodesData.Nodes;
-    return checkNodeQueue(5);
+    return checkNodeQueue(10);
```

Comment:
Same comment as above. This number of execution nodes to be read from environment variables.